pkg/policy/policies.go

```diff
@@ -132,11 +149,49 @@ func (ps *Policies) set(id int, p *Policy) error {
 	ps.policiesArray[id] = p
 	ps.filterEnabledPoliciesMap[p] = id
 
-	ps.Compute()
+	ps.compute()
 
 	return nil
 }
 
+// TODO: runtime API should encapsulate the following calls:
```

Comment:
This API call will be protected by the policies mutex, right?

Reply:
Yes, it will. @josedonizetti those are the current steps available in this PR for the runtime API.

Reply:
As we have discussed, we're going to move all Events and Policies into PolicyManager.